Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -333,6 +343,11 @@ namespace AzToolsFramework::ViewportUi::Internal
         m_viewportBorderBackButton.hide();
     }
 
+    inline const bool ViewportUiDisplay::GetViewportBorderVisible() const
```

Comment:
update: This does not need to be marked `inline` in the .cpp (only if it's in the header) and you can remove the const from the the bool return true (`const bool` -> `bool`)